client/main.js

```diff
@@ -0,0 +1,22 @@
+
+const getSentenceButton = document.getElementById("get-sentence-btn")
+const sentenceContainer = document.getElementById("sentence-container")
+
+const serverAddress = "localhost"
+const serverPort = 3000
+const getSentenceRoute = "greateSentence"
+
+const getSentenceURI = `http://${serverAddress}:${serverPort}/${getSentenceRoute}`
+
+function getSentenceFromServer() {
```

Comment:
Using an async function will make the code more readable

```suggestion
async function getSentenceFromServer() {

 const sentence = await fetch(new Request(getSentenceURI)).then(res => res.text());
setSentence(text)
}
```

Reply:
nice